README.md

```diff
@@ -206,7 +206,7 @@ func main() {
 		AuthURL:  fmt.Sprintf("%s/rest/auth/1/session", base),
 	}
 
-	jiraClient, err := jira.NewClient(tp.Client(), base)
+	jiraClient, _ := jira.NewClient(tp.Client(), base)
```

Comment:
You dropped `err`, because it is not used in the example, right?
If yes, i would go for an additional 
```go
if err != nil {
	panic(err)
}
```
here, because it is an example and is _really_ often used as a copy / paste go to start code.
With the additional `panic`, people might start think about error handling.

Reply:
Sure, I will return it